includes/vendor/myparcelnl/sdk/src/Model/Consignment/AbstractConsignment.php

```diff
@@ -394,6 +408,14 @@ public function __construct()
         $this->helper = new Helpers();
     }
 
+    /**
+     * @return array
+     */
+    public function getInsurancePossibilities()
+    {
+        return $this->insurance_possibilities_local;
```

Comment:
insurance_possibilities_local kan niet gebruikt worden voor andere landen toch? Dus dit kan alleen terug gegeven worden als consignment local is. Toch?

Reply:
done